django/cantusdb_project/main_app/management/commands/add_differentiae.py

```diff
@@ -0,0 +1,89 @@
+from main_app.models import Chant, Differentia
+from django.core.management.base import BaseCommand
+
+
+# Python script to sync the differentia database with CantusDB. Previously, we had a CharField that represented
+# the differentia ID. Now, we change this field to a ForeignKey that points to the new Differentia model that contains
+# the ID, melodic transcription, and mode.
+
+
+DIFFERENTIA_DATA = "differentia_data.txt"
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        differentia_database = get_differentia_data(DIFFERENTIA_DATA)
+        total = len(differentia_database)
+        count = 0
+        for item in differentia_database:
+            differentia_id, melodic_transcription, mode, _ = item
+            differentia = Differentia.objects.filter(
+                differentia_id=differentia_id
+            ).first()
```

Comment:
can we just use `Differentia.objects.get()` here? If we somehow end up with two differentiae with the same differentia_id, I'd want an exception to be raised, rather than just picking one arbitrarily.